recipes/base_config.rb

```diff
@@ -15,6 +15,7 @@
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
+include_recipe "aws-parallelcluster::setup_envars"
```

Comment:
is this not needed for base_install as well?

Reply:
`base_install` is always called by `base_config`, so it inherits the values of the envars.

Reply:
Btw, I think you're right: we should add `setup_envars` because there are execution paths where only one of them is executed.